resources/measures/HPXMLtoOpenStudio/resources/materials.rb

```diff
@@ -279,11 +301,11 @@ def self.RoofingAsphaltShinglesMed
   end
 
   def self.RoofingAsphaltShinglesLight
-    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 0.8)
+    return self.RoofMaterial('Asphalt Shingles, Light', 0.91, 0.75)
```

Comment:
Can we use the same map here?

Reply:
Absorptance for roofing materials are dependent on both material (shingle) and color (light) in restructure-v3, so the exterior wall map isn't applicable and hard-coding was just as efficient. https://github.com/NREL/resstock/blob/restructure-v3/resources/hpxml-measures/HPXMLtoOpenStudio/resources/constructions.rb#L1135